proxy/envoy/testdata/envoy.json.golden

```diff
@@ -2,7 +2,7 @@
   "listeners": [],
   "lds": {
     "cluster": "lds",
-    "refresh_delay_ms": 10
+    "refresh_delay_ms": 10000
```

Comment:
where did the default change?

Reply:
These are fake values, but 10ms does not validate, so I bumped it up.